### ISSUE TITLE
Can only ever get latest package from pip

### ISSUE THREAD

**Issue by oharaandrew314** [novemberfiveco/s3pypi#29]
Once I upload a new version of a package to my s3pypi, then I can only ever get the latest version after that.  Looking at the repo in s3, I can see the entire history of `tar.gz` files, but inspecting the `html` file lists only the most recent version.

```
$ pip install pcns-common==0.9 --extra-index-url <my-repo-url>
Could not find a version that satisfies the requirement pcns-common==0.9 (from versions: 0.10)
No matching distribution found for pcns-common==0.9
```

Is this a problem with `s3pypi` or my own `setup.py`?

**Comment by mdwint** [comment]
There used to be a bug in s3pypi that caused this. Are you using the latest version?

```pip install --upgrade s3pypi```

**Comment by oharaandrew314** [comment]
Yes, I get the latest version every time I push.

**Comment by mdwint** [comment]
You should use [semantic versioning](http://semver.org/) in setup.py. Try this instead:
```
version='0.10.0',
```

I'll add an error message instead of silently discarding index.html.

**Comment by oharaandrew314** [comment]
Yes, using a semantic versioning scheme seems to have fixed the problem.  I now see all versions in the index.html that use this scheme, and I can download them all with pip.

An error message would be very helpful.  Thanks for the help!
